versions/v6/templates/story/default.php

```diff
@@ -6,7 +6,10 @@
 		<?php echo display_story_header_contents( $post ); ?>
 
 		<div class="row content-wrap">
-			<div class="col-lg-10 col-md-10 offset-lg-1 offset-md-1">
+			<div class="col-lg-2 col-md-2">
```

Comment:
Flip the positions of these columns so that related stories display underneath the article on mobile (and so that keyboard users don't have to tab over these links before getting to the article content).  Add `.flex-lg-first` to set ideal column ordering for desktop.

I'd also suggest adding `.mb-5` to the column surrounding the story content to help give some breathing room between it and the related stories sections on mobile.